### DISCUSSION TITLE
🍔 Fat is good 🍔

### DISCUSSION THREAD

**Discussion by LivingWithHippos** [LivingWithHippos/unchained-android#138] [Feedback]
### 🏷️ New features:

- [x] added connectivity check
- [x] two new search plugins
- [x] added multi select to download lists
- [x] added automatic crash reporter (debug APK version only)
- [x] added  download all button in folder screen

### 🦟 Bug fixes:

- [x] search icon does not disappear anymore
- [x] delete all button in download list now works

### 🔎  Enhancements:

- [x] sorting is working better and has been added to the search screen
- [x] downloads to phone works better
- [x] merged the new download and download list tabs, since they did the same things (use the new fab in the list screen)

Notes:

- as you can see, there are two APKs again. The debug version (bigger one) will now report info to my server when the app crashes. There are no advantages over the normal APK, but please use it if you get crashes
- the Play Store release will be delayed a little since I need to collect feedback (many changes this time). I've added a discussion in the Discussions tab

<hr /><em>This discussion was created from the release <a href='https://github.com/LivingWithHippos/unchained-android/releases/tag/4.16.4-beta'>🍔 Fat is good 🍔</a>.</em>

**Comment by anon-y** [comment]
This is such an amazing app! Thank you for taking the time to write this. 

edit: nm saw 2 new sources! Wow this is incredible!

**Reply by LivingWithHippos** [reply]
Of course! Actually if you know html and regexs you can even write and install them yourself. I'll be making some changes to the search engine soon to make it compatible with more websites, so I'd wait before its 2.0 version is out.

You can also just open an issue like  #140 and ask for a certain website, I just need a link. Not all search websites can be parsed but most can.

**Reply by anon-y** [reply]
> Of course! Actually if you know html and regexs you can even write and install them yourself. I'll be making some changes to the search engine soon to make it compatible with more websites, so I'd wait before its 2.0 version is out.
> 
> You can also just open an issue like  #140 and ask for a certain website, I just need a link. Not all search websites can be parsed but most can.

Thank you! 2.0 sounds spicy 🔥

**Reply by LivingWithHippos** [reply]
> Thank you! 2.0 sounds spicy 🔥

Well it's just an engine update that will allow me to add more websites that cannot be added right now 😄. It's 2.0 because it will break compatibility with the 1.x addons (I'll also update those)

**Reply by BlueCloudCaddo** [reply]
Im having trouble installing ther 2 new plugins... (I probably just forgot HOW I installed the prior ones. ) Can you walk me through?  When I try to share  .Unchained files with the app, its not listed.

**Reply by BlueCloudCaddo** [reply]
Love it nonetheless!!!

**Reply by LivingWithHippos** [reply]
Thanks! I made some modifications on the plugin so the plugins are for the search engine 2 while the  published app has search engine 1. I had no time to publish the new version, next time I'll change the way I publish stuff so it won't happen. You can download the latest nightly or wait for the next update (it should come out today or tomorrow)

**Reply by LivingWithHippos** [reply]
A new version is out, you can wait for that to appear in the play store or download it from the release page of github in the meanwhile
